torch/nn/functional.py

```diff
@@ -1925,6 +1925,22 @@ def hardsigmoid(input: Tensor, inplace: bool = False) -> Tensor:
     return torch._C._nn.hardsigmoid(input)
 
 
+def bias(input: Tensor, bias: Tensor) -> Tensor:
+    r"""
+    Applies a bias transformation to the incoming data: :math:`y = x + b`.
+
+    Shape:
+
+        - Input: :math:`(*, num\_features)` where `*` means any number of
+          additional dimensions, including none
+        - Bias: :math:`(num\_features)`
+        - Output: :math:`(*, num\_features)`
+    """
+    if has_torch_function_variadic(input, bias):
+        return handle_torch_function(bias, (input, bias), input, bias)
```

Comment:
I think `bias` is ambiguous here - is it the Tensor input or the function itself? In the call to `handle_torch_function()`, both should be referenced, so I expected to see a failing test related to torch_function handling.

The usual way to get around this is to name the input something slightly different like `bias_`.